script/reverse_urls.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+from urllib.parse import urlsplit
+import re
+
+def reverse_url(url):
```

Comment:
Can we add a doctest showcasing what this does? Something like:
```
"""
>>> http://some.url/to?reverse=true
RESULT
"""
```
It's not clear right away what it means and the body is not trivial, so it's good to have a doc+test for clarity.

Reply:
Added